extensions/kubernetes-client/spi/src/main/java/io/quarkus/kubernetes/client/spi/KubernetesDevServiceInfoBuildItem.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.kubernetes.client.spi;
+
+import org.testcontainers.containers.GenericContainer;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class KubernetesDevServiceInfoBuildItem extends SimpleBuildItem {
+    private final String kubeConfig;
+    private final GenericContainer container;
```

Comment:
Let's not bring in the GenericContainer object, as we want to limit the number of dependencies we introduce in the spi module. I think that this could possibly be replaced with host / port.

Reply:
That works using this code with a DummyContainer implementing Closeable at the consumer side. It has been tested successfully and kind container was well killed
```
        return new DevServicesResultBuildItem.RunningDevService(
            ArgocdProcessor.FEATURE,
            kubeServiceInfo.getContainer().getContainerId(),
            new ContainerShutdownCloseable(new DummyContainer(), ArgocdProcessor.FEATURE),
            configOverrides).toBuildItem();
    }

    private class DummyContainer extends GenericContainer<DummyContainer> implements Closeable {
        private static final Logger LOG = Logger.getLogger(DummyContainer.class);
        @Override
        public void close() {
            LOG.info("Closing the argocd container ...");
        }
    }
```